Nodejs/Product/Nodejs/Debugger/NodeDebugger.cs

```diff
@@ -530,15 +530,13 @@ public NodeDebugger(Uri debuggerEndpointUri, int id)
             foreach (NodeModule module in modules) {
                 NodeModule newModule;
                 if (GetOrAddModule(module, out newModule)) {
-                    if (newModule.FileName != newModule.JavaScriptFileName) {
```

Comment:
Hmm - did you try loading a TS project? This would probably evaluate to true in those scenarios.

I suspect there might be a better way of resolving the issue, but I don't think it'll *break* anything to add this (other than having a perf impact) - basically what you're doing is attempting to bind the breakpoints a little later than we normally do (after scripts have been loaded), in which case I don't understand how the only thing that's changed with 6.x is the case sensitivity, and I think it would be worth digging into it a little further to understand specifically what changed or what existing issue has manifested itself (and also if there's a way to log this on our end so that it's easier to debug in the future.)

Also worth looking into when VSCode binds breakpoints, and if there's a separate codepath for JS vs. source-mapped files.